src/interfaces/protocols-query.ts

```diff
@@ -24,7 +24,9 @@ export class ProtocolsQuery extends Message<ProtocolsQueryMessage> {
 
 
   public static async parse(message: ProtocolsQueryMessage): Promise<ProtocolsQuery> {
```

Comment:
A couple lines above here, I added `readonly author!: string;` since `ProtocolsQuery` had previously required auth. We should remove that line now.

Reply:
Nice catch! Removd.